resources/global/css/components/_buttons.scss

```diff
@@ -179,6 +179,38 @@
   border-color: $black !important;
 }
 
+.btn-tertiary {
```

Comment:
One for the dirt doc, but will note here too - we have both resources/global/.... and resouces/assets/...

resources/assets was what we always had from the beginning of Laravel app.  In recent rebrand work, WCD added in resources/global for styles that would also be shared by Discourse and MediaWiki, but resources/assets/ kept for anything Laravel specific.  I'm not sure how useful this distinction is and I've personally found it a bit confusing at times - some things are in two places.  To be revisited...

Reply:
Understood.  If we go down the Vue route then active CSS will gradually migrate out of the global files.